crypto/mem_clr.c

```diff
@@ -60,6 +60,7 @@
 #include <string.h>
 #include <openssl/crypto.h>
 
+extern unsigned char cleanse_ctr;
```

Comment:
Got it! Thanks. Well, warning is a little bit masochistic for my personal taste, but ok, it's surely sensible at some level...